### PR TITLE
Migrate OneNote Add-ins page

### PR DESCRIPTION
**Pull request by KarenYin01**
Migrate the original page to new designed landing page. Refer to [Dev Task 231648](https://dev.azure.com/ceapex/Engineering/_workitems/edit/231648).

Here is the guidance of new designed landing page.
https://aka.ms/docslanding

### CODE REVIEW

**Comment by o365devx** [comment]
Docs Build status updates of commit _[6d23261](https://github.com/KarenYin01/office-js-docs-pr/commits/6d23261be3faad85a80f134e96fde97632fc275e)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/index.md](https://github.com/KarenYin01/office-js-docs-pr/blob/Migrate-OneNote-Add-ins-page/docs/index.md) | :warning:Warning | | [Details](#user-content-d93b521a-85a70f57)
[docs/reference/overview/onenote-add-ins-javascript-reference.md](https://github.com/KarenYin01/office-js-docs-pr/blob/Migrate-OneNote-Add-ins-page/docs/reference/overview/onenote-add-ins-javascript-reference.md) | :warning:Warning | | [Details](#user-content-8469a05e-85a70f57)
[docs/toc.yml](https://github.com/KarenYin01/office-js-docs-pr/blob/Migrate-OneNote-Add-ins-page/docs/toc.yml) | :warning:Warning | | [Details](#user-content-a2f50e64-85a70f57)
[docs/onenote/index.md](https://github.com/KarenYin01/office-js-docs-pr/blob/Migrate-OneNote-Add-ins-page/docs/onenote/index.md) | :white_check_mark:Succeeded | |
[docs/onenote/index.yml](https://github.com/KarenYin01/office-js-docs-pr/blob/Migrate-OneNote-Add-ins-page/docs/onenote/index.yml) | :white_check_mark:Succeeded | |

<a id="d93b521a-85a70f57"></a>
### [docs/index.md](https://github.com/KarenYin01/office-js-docs-pr/blob/Migrate-OneNote-Add-ins-page/docs/index.md)
 - **Line 0**: **[Warning-[InvalidFileLink](https://review.docs.microsoft.com/en-us/help/contribute/contribute-how-to-write-links?branch=master)]** `````Invalid file link:(~/onenote/index.md).`````

<a id="8469a05e-85a70f57"></a>
### [docs/reference/overview/onenote-add-ins-javascript-reference.md](https://github.com/KarenYin01/office-js-docs-pr/blob/Migrate-OneNote-Add-ins-page/docs/reference/overview/onenote-add-ins-javascript-reference.md)
 - **Line 35**: **[Warning-[InvalidFileLink](https://review.docs.microsoft.com/en-us/help/contribute/contribute-how-to-write-links?branch=master)]** `````Invalid file link:(~/onenote/index.md).`````

<a id="a2f50e64-85a70f57"></a>
### [docs/toc.yml](https://github.com/KarenYin01/office-js-docs-pr/blob/Migrate-OneNote-Add-ins-page/docs/toc.yml)
 - **Line 0**: **[Warning-[InvalidFileLink](https://review.docs.microsoft.com/en-us/help/contribute/contribute-how-to-write-links?branch=master)]** `````Invalid file link:(~/onenote/index.md).`````

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C6%5C8%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202006080801528458-1930%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=iE2Jj9b4YWL5TRhq%2Fn804Jpti3sTsXbaOTNeJEuEhZw%3D&st=2020-06-08T07%3A58%3A05Z&se=2020-07-09T08%3A03%3A05Z&sp=r).

If you see build warnings/errors with permission issues, it might be due to single sign-on (SSO) enabled on Microsoft's GitHub organizations. Please follow instructions [here](https://teams.microsoft.com/l/message/19:7ecffca1166a4a3986fed528cf0870ee@thread.skype/1590030602688?tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47&groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&parentMessageId=1590030602688&teamName=Docs%20Support&channelName=General&createdTime=1590030602688) to re-authorize your GitHub account to Docs Build.

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/57856ef7-e8cf-54ca-0d12-df5f42d1e8bb?tabName=brokenlinks).

**Note:** Your PR may contain errors or warnings unrelated to the files you changed. This happens when external dependencies like GitHub alias, Microsoft alias, cross repo links are updated. Please use these [instructions](https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master&tabs=pr-validation-error-solution#q-why-do-i-get-pr-validation-result-with-cryptic-error-or-warning-messages-not-related-to-the-changes-i-make-in-my-working-branch) to resolve them.

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by o365devx** [comment]
Docs Build status updates of commit _[d9af3ec](https://github.com/KarenYin01/office-js-docs-pr/commits/d9af3ec18f657827b1763d6c95340e1895cf85c4)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/index.md](https://github.com/KarenYin01/office-js-docs-pr/blob/Migrate-OneNote-Add-ins-page/docs/index.md) | :warning:Warning | | [Details](#user-content-d93b521a-9f9b8404)
[docs/reference/overview/onenote-add-ins-javascript-reference.md](https://github.com/KarenYin01/office-js-docs-pr/blob/Migrate-OneNote-Add-ins-page/docs/reference/overview/onenote-add-ins-javascript-reference.md) | :warning:Warning | | [Details](#user-content-8469a05e-9f9b8404)
[docs/onenote/index.md](https://github.com/KarenYin01/office-js-docs-pr/blob/Migrate-OneNote-Add-ins-page/docs/onenote/index.md) | :white_check_mark:Succeeded | |
[docs/onenote/index.yml](https://github.com/KarenYin01/office-js-docs-pr/blob/Migrate-OneNote-Add-ins-page/docs/onenote/index.yml) | :white_check_mark:Succeeded | |
[docs/toc.yml](https://github.com/KarenYin01/office-js-docs-pr/blob/Migrate-OneNote-Add-ins-page/docs/toc.yml) | :white_check_mark:Succeeded | |

<a id="d93b521a-9f9b8404"></a>
### [docs/index.md](https://github.com/KarenYin01/office-js-docs-pr/blob/Migrate-OneNote-Add-ins-page/docs/index.md)
 - **Line 0**: **[Warning-[InvalidFileLink](https://review.docs.microsoft.com/en-us/help/contribute/contribute-how-to-write-links?branch=master)]** `````Invalid file link:(~/onenote/index.md).`````

<a id="8469a05e-9f9b8404"></a>
### [docs/reference/overview/onenote-add-ins-javascript-reference.md](https://github.com/KarenYin01/office-js-docs-pr/blob/Migrate-OneNote-Add-ins-page/docs/reference/overview/onenote-add-ins-javascript-reference.md)
 - **Line 35**: **[Warning-[InvalidFileLink](https://review.docs.microsoft.com/en-us/help/contribute/contribute-how-to-write-links?branch=master)]** `````Invalid file link:(~/onenote/index.md).`````

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C6%5C8%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202006080808222389-1930%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=a0q6Eo%2F%2FQuN7yePHyQeDQpNx%2FVEz24kDI%2F%2BejCAQS1A%3D&st=2020-06-08T08%3A05%3A03Z&se=2020-07-09T08%3A10%3A03Z&sp=r).

If you see build warnings/errors with permission issues, it might be due to single sign-on (SSO) enabled on Microsoft's GitHub organizations. Please follow instructions [here](https://teams.microsoft.com/l/message/19:7ecffca1166a4a3986fed528cf0870ee@thread.skype/1590030602688?tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47&groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&parentMessageId=1590030602688&teamName=Docs%20Support&channelName=General&createdTime=1590030602688) to re-authorize your GitHub account to Docs Build.

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/57856ef7-e8cf-54ca-0d12-df5f42d1e8bb?tabName=brokenlinks).

**Note:** Your PR may contain errors or warnings unrelated to the files you changed. This happens when external dependencies like GitHub alias, Microsoft alias, cross repo links are updated. Please use these [instructions](https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master&tabs=pr-validation-error-solution#q-why-do-i-get-pr-validation-result-with-cryptic-error-or-warning-messages-not-related-to-the-changes-i-make-in-my-working-branch) to resolve them.

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by o365devx** [comment]
Docs Build status updates of commit _[8532556](https://github.com/KarenYin01/office-js-docs-pr/commits/8532556ff618f132cc4e465e287d21586fb35a49)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/index.md](https://github.com/KarenYin01/office-js-docs-pr/blob/Migrate-OneNote-Add-ins-page/docs/index.md) | :warning:Warning | | [Details](#user-content-d93b521a-18ef4873)
[docs/onenote/index.md](https://github.com/KarenYin01/office-js-docs-pr/blob/Migrate-OneNote-Add-ins-page/docs/onenote/index.md) | :white_check_mark:Succeeded | |
[docs/onenote/index.yml](https://github.com/KarenYin01/office-js-docs-pr/blob/Migrate-OneNote-Add-ins-page/docs/onenote/index.yml) | :white_check_mark:Succeeded | |
[docs/reference/overview/onenote-add-ins-javascript-reference.md](https://github.com/KarenYin01/office-js-docs-pr/blob/Migrate-OneNote-Add-ins-page/docs/reference/overview/onenote-add-ins-javascript-reference.md) | :white_check_mark:Succeeded | |
[docs/toc.yml](https://github.com/KarenYin01/office-js-docs-pr/blob/Migrate-OneNote-Add-ins-page/docs/toc.yml) | :white_check_mark:Succeeded | |

<a id="d93b521a-18ef4873"></a>
### [docs/index.md](https://github.com/KarenYin01/office-js-docs-pr/blob/Migrate-OneNote-Add-ins-page/docs/index.md)
 - **Line 0**: **[Warning-[InvalidFileLink](https://review.docs.microsoft.com/en-us/help/contribute/contribute-how-to-write-links?branch=master)]** `````Invalid file link:(~/onenote/index.md).`````

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C6%5C8%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202006080809111871-1930%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=o9xOwgbDrbagX4EnRhY4SkZZIgfvOs84lQQyYUcE9mE%3D&st=2020-06-08T08%3A06%3A41Z&se=2020-07-09T08%3A11%3A41Z&sp=r).

If you see build warnings/errors with permission issues, it might be due to single sign-on (SSO) enabled on Microsoft's GitHub organizations. Please follow instructions [here](https://teams.microsoft.com/l/message/19:7ecffca1166a4a3986fed528cf0870ee@thread.skype/1590030602688?tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47&groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&parentMessageId=1590030602688&teamName=Docs%20Support&channelName=General&createdTime=1590030602688) to re-authorize your GitHub account to Docs Build.

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/57856ef7-e8cf-54ca-0d12-df5f42d1e8bb?tabName=brokenlinks).

**Note:** Your PR may contain errors or warnings unrelated to the files you changed. This happens when external dependencies like GitHub alias, Microsoft alias, cross repo links are updated. Please use these [instructions](https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master&tabs=pr-validation-error-solution#q-why-do-i-get-pr-validation-result-with-cryptic-error-or-warning-messages-not-related-to-the-changes-i-make-in-my-working-branch) to resolve them.

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by ElizabethSamuel-MSFT** [comment]
Closing and reopening to generate preview site.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[8532556](https://github.com/KarenYin01/office-js-docs-pr/commits/8532556ff618f132cc4e465e287d21586fb35a49)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/index.md](https://github.com/KarenYin01/office-js-docs-pr/blob/Migrate-OneNote-Add-ins-page/docs/index.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/index?branch=pr-en-us-1930) | [Details](#user-content-d93b521a-aa86b999)
[docs/onenote/index.md](https://github.com/KarenYin01/office-js-docs-pr/blob/Migrate-OneNote-Add-ins-page/docs/onenote/index.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/index?branch=pr-en-us-1930) |
[docs/onenote/index.yml](https://github.com/KarenYin01/office-js-docs-pr/blob/Migrate-OneNote-Add-ins-page/docs/onenote/index.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/onenote/index?branch=pr-en-us-1930) |
[docs/reference/overview/onenote-add-ins-javascript-reference.md](https://github.com/KarenYin01/office-js-docs-pr/blob/Migrate-OneNote-Add-ins-page/docs/reference/overview/onenote-add-ins-javascript-reference.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/reference/overview/onenote-add-ins-javascript-reference?branch=pr-en-us-1930) |
[docs/toc.yml](https://github.com/KarenYin01/office-js-docs-pr/blob/Migrate-OneNote-Add-ins-page/docs/toc.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/onenote/index?branch=pr-en-us-1930) |

<a id="d93b521a-aa86b999"></a>
### [docs/index.md](https://github.com/KarenYin01/office-js-docs-pr/blob/Migrate-OneNote-Add-ins-page/docs/index.md)
 - **Line 0**: **[Warning-[InvalidFileLink](https://review.docs.microsoft.com/en-us/help/contribute/contribute-how-to-write-links?branch=master)]** `````Invalid file link:(~/onenote/index.md).`````

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C6%5C8%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202006081628043580-1930%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=QGVwuHYoKE4aeygpKtAXslZwXYUqN8zGmbCHcY%2FrZi0%3D&st=2020-06-08T16%3A24%3A47Z&se=2020-07-09T16%3A29%3A47Z&sp=r).

If you see build warnings/errors with permission issues, it might be due to single sign-on (SSO) enabled on Microsoft's GitHub organizations. Please follow instructions [here](https://teams.microsoft.com/l/message/19:7ecffca1166a4a3986fed528cf0870ee@thread.skype/1590030602688?tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47&groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&parentMessageId=1590030602688&teamName=Docs%20Support&channelName=General&createdTime=1590030602688) to re-authorize your GitHub account to Docs Build.

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/57856ef7-e8cf-54ca-0d12-df5f42d1e8bb?tabName=brokenlinks).

**Note:** Your PR may contain errors or warnings unrelated to the files you changed. This happens when external dependencies like GitHub alias, Microsoft alias, cross repo links are updated. Please use these [instructions](https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master&tabs=pr-validation-error-solution#q-why-do-i-get-pr-validation-result-with-cryptic-error-or-warning-messages-not-related-to-the-changes-i-make-in-my-working-branch) to resolve them.

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

File: docs/onenote/index.yml [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+title: OneNote Add-ins documentation # < 60 chars
+summary: With OneNote add-ins, you can use familiar web technologies such as HTML, CSS, and JavaScript to build a solution that can run in OneNote on the web. # < 160 chars
+
+metadata:
+  title: OneNote Add-ins documentation # Required; page title displayed in search results. Include the brand. < 60 chars.
+  description: Resources for learning about OneNote add-ins. # Required; article description that is displayed in search results. < 160 chars.
+  ms.service: o365-proplus-itpro #Required; service per approved list. service slug assigned to your service by ACOM.
+  ms.topic: landing-page # Required
+  ms.date: 07/05/2019 #Required; mm/dd/yyyy format.
```
Comment (Rick-Kirkham) [inline docs/onenote/index.yml:11]: update ms.date

**Pull request merged**